meta-beluga/recipes-kernel/modules/linux-wlan-modules-beluga_p.bb

```diff
@@ -23,6 +23,6 @@ SYSTEMD_PACKAGES = "${PN}"
 SYSTEMD_SERVICE:${PN} = "wlan-module-load.service"
 
 do_install:append() {
-    install -d ${D}/lib/systemd/system/
-    cp ${WORKDIR}/wlan-module-load.service ${D}/lib/systemd/system/
+    install -d ${D}${systemd_system_unitdir}
```

Comment:
Probably a good fix using `install` instead of `cp`, but I can't find documentation on ${systemd_system_unitdir} on yoctoproject.org.

Reply:
See previous comment ;)